.eslintrc.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+	extends: ['eslint:recommended', 'prettier', 'prettier/react'],
+	plugins: ['react', 'prettier'],
+	parser: 'babel-eslint',
+	parserOptions: {
+		ecmaVersion: 2016,
+		sourceType: 'module',
+		ecmaFeatures: {
+			jsx: true
+		}
+	},
+	env: {
+		browser: true,
+		node: true,
+		es6: true
+	},
+	rules: {
+		curly: ['error', 'all'],
+		'prettier/prettier': [
+			'error',
+			{
+				printWidth: 120,
+				semi: false,
+				singleQuote: true,
+				trailingComma: 'none', // ideally 'all'...
```

Comment:
`trailingComma: 'all'` results in a large initial diff, but smaller diffs later on, as adding things to hashes/arrays doesn't require you to add a comma to the end of the previous line.